build.gradle

```diff
@@ -24,5 +24,6 @@ allprojects {
         maven {
             url 'file:///tmp/maven/'
         }
+        mavenLocal()
```

Comment:
This is here for testing against a local release - will be removed after v3 core release